content/_guides/faq.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Frequently Asked Questions
+category: about-irc
+---
+
+- TOC
+{:toc}
+
+## You need to identify via SASL to use this server
+
+You have tried to connect from a
+[SASL access only range](guides/sasl#sasl-access-only-ip-ranges)
+
+## Cannot send to nick/channel
+
+## Cannot change nickname while banned on channel
```

Comment:
Could we make sub-points be ### for a better hierarchy?

Reply:
that was to provide 2 headings for the same text, not a hierarchy thing.